frontend/src/metabase/query_builder/components/expressions/ExpressionEditorTextfield/ExpressionEditorTextfield.tsx

```diff
@@ -40,6 +45,50 @@ import {
 ace.config.set("basePath", "/assets/ui/");
 ace.config.set("useStrictCSP", true);
 
+export type SuggestionFooter = {
+  footer: true;
+  name: string;
+  icon: IconName;
+  href: string;
+};
+
+type SuggestWithFooters = {
+  suggestions: (Suggestion | SuggestionFooter)[];
+  helpText?: HelpText;
+};
+
+function suggestWithFooters(
+  args: SuggestArgs,
+  showMetabaseLinks: boolean,
+): SuggestWithFooters {
+  const res = suggest(args);
+
+  const suggestions: (Suggestion | SuggestionFooter)[] = res.suggestions ?? [];
+
+  if (showMetabaseLinks) {
+    if (args.startRule === "aggregation") {
+      suggestions?.push({
+        footer: true,
+        name: t`View all aggregations`,
+        icon: "external",
+        href: "https://www.metabase.com/docs/latest/questions/query-builder/expressions-list#aggregations",
+      });
+    } else {
```

Comment:
Should we check `args.startRule === "function"` just to be sure?

Reply:
We want to render the footer for all other cases so I think we should leave it as is. That way, adding more cases won't break this).